bld/namelist_files/namelist_definition_mpassi.xml

```diff
@@ -1,3 +1,4 @@
+dazlich@derecho4:~/ewv2.1/mpassi-pres/components/mpas-seaice> cat bld/namelist_files/namelist_definition_mpassi.xml
```

Comment:
Need to remove this

Reply:
oops, fixed. That's what I get for editing in GitHub.

Reply:
Yep, addressed by [eb5cf9f](https://github.com/EarthWorksOrg/mpas-seaice/pull/8/commits/eb5cf9f1fa424cb809bc985cb197353f0ef204df)